modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -0,0 +1,860 @@
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT RENAME USER
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| ALTER USER
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+ALTER USER name [IF EXISTS]
+  [SET [PLAINTEXT \| ENCRYPTED] PASSWORD 'password']
+  [[SET PASSWORD] CHANGE [NOT] REQUIRED]
+  [SET STATUS {ACTIVE \| SUSPENDED} ]
+  [SET HOME DATABASE name]
+  [REMOVE HOME DATABASE]
+----
+
+| Description
+a|
+Modifies the settings for an existing user.
+At least one `SET` or `REMOVE` clause is required.
+`SET` and `REMOVE` clauses cannot be combined in the same command.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-alter-users[Modifying users].
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT SET PASSWORD
+----
+
+[source, privilege, role="noheader"
+----
+GRANT SET USER STATUS
+----
+
+[source, privilege, role="noheader"]
+----
+GRANT SET USER HOME DATABASE
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+
+[cols="<15s,<85"]
+|===
+
+| Command
+m| ALTER CURRENT USER SET PASSWORD
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+ALTER CURRENT USER SET PASSWORD FROM 'oldPassword' TO 'newPassword'
+----
+
+| Description
+a|
+Changes the current user's password.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-alter-password[Changing the current user's password].
+
+| Required privilege
+a| None
+
+|===
+
+
+[cols="<15s,<85"]
+|===
+
+| Command
+m| DROP USER
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+DROP USER name [IF EXISTS]
+----
+
+| Description
+a|
+Removes an existing user.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-drop-users[Delete users].
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT DROP USER
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+
+[NOTE]
+====
+The `SHOW USER[S] PRIVILEGES` command is only available in Neo4j Enterprise Edition. label:enterprise-edition[]
+====
+
+
+[[access-control-current-users]]
+== Listing current user
+
+The currently logged-in user can be seen using `SHOW CURRENT USER`, which will produce a table with the following columns:
+
+[options="header", width="100%", cols="2a,4,2m,^.^,^.^"]
+|===
+| Column
+| Description
+| Type
+| Community Edition
+| Enterprise Edition
+
+| user
+| User name
+| STRING
+| {check-mark}
+| {check-mark}
+
+| roles
+| Roles granted to the user.
+
+Will return `null` in community edition.
+| LIST OF STRING
```

Comment:
I guess a PR that switches from old to new type syntax needs to be ported over to this PR

Reply:
I have only updated the links on these pages.

Reply:
I'll assume it will be updated once additional PRs are ported afterwards but will leave the comment open to remember to check that if that's ok with you?